docs/controls/SwitchPresenter.md

```diff
@@ -0,0 +1,74 @@
+---
+title: SwitchPresenter XAML Control
+author: michael-hawker
+description: A XAML ContentPresenter which can act like a switch statement for showing different UI based on a condition.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, switchpresenter, contentpresenter, visibility
+dev_langs:
+  - csharp
+---
+
+# SwitchPresenter XAML Control
+
+<!-- Describe your control -->
+The [SwitchPresenter](/dotnet/api/microsoft.toolkit.uwp.ui.controls.switchpresenter) control acts like a switch statement for XAML. It allows a developer to display certain content based on the condition of another value as an alternative to managing multiple Visibility values or complex visual states.
+
+Unlike traditional approaches of showing/hiding components within a page, the `SwitchPresenter` will only load and attach the matching Case's content to the Visual Tree.
+
+> [!div class="nextstepaction"]
+> [Try it in the sample app](uwpct://controls?sample=SwitchPresenter)
+
+## Example
+
+```xaml
+    <controls:SwitchPresenter Value="{Binding SelectedItem, ElementName=Lookup}">
+      <controls:Case Value="Confirmation Code">
+        <StackPanel>
+          <TextBox Name="ConfirmationCodeValidator"
+                   ui:TextBoxExtensions.Regex="^[a-zA-Z]{6}$"
+                   Header="Confirmation code"
+                   PlaceholderText="6 letters" />
+          <TextBlock Visibility="{Binding (ui:TextBoxExtensions.IsValid), ElementName=ConfirmationCodeValidator}">Thanks for entering a valid code!</TextBlock>
```

Comment:
Should update to use the Text property in all these cases here and in the sample.

Reply:
```suggestion
          <TextBlock Visibility="{Binding (ui:TextBoxExtensions.IsValid), ElementName=ConfirmationCodeValidator}" Text="Thanks for entering a valid code!" />
```